jupyter/analysis-automation/BasicExample.ipynb

```diff
@@ -0,0 +1,271 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import requests\n",
+    "import asyncio\n",
+    "from systemlink.clients.nitdmreader.api.metadata_api import MetadataApi\n",
+    "from systemlink.clients.nitdmreader.models.file_list import FileList\n",
+    "\n",
+    "metadata_api = MetadataApi()\n",
+    "import scrapbook as sb\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
```

Comment:
numpy is imported but never used. Maybe it makes sense to use numpy to calculate the maximum/minimum.